src/react-components/src/components/02-molecules/Cards/EditionCard.tsx

```diff
@@ -0,0 +1,66 @@
+/* MC-14 Edition Card */
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import Image from "../../01-atoms/Images/Image/Image";
+import BasicLink from "../../01-atoms/Links/BasicLink";
+
+
+export interface EditionCardProps {
+  id: string;
+  blockName?: string;
+
+  coverUrl: string;
+
+  editionHeadingElement: JSX.Element;
+
+  editionInfo: string[];
+
+  readOnlineLink?: string;
+  downloadLink?: string;
+}
+
+export default class EditionCard extends React.Component<EditionCardProps, {}> {
```

Comment:
You should start incorporating some documentation to the components in typedoc or esdoc notation. So in this case, it'll be like:
```
/**
 * EditionCard component that renders information for an edition.
 */
export default class EditionCard...
```

I was going to comment that it would be useful if the component was more generic and allowed more than one online or download link, mostly because this is true for items in SimplyE that can have multiple epub files, pdf files, or other links. But it makes sense to have a specific component where you know it will always have at most two links. In cases like that, I think it could use documentation to make that clear.

Reply:
I agree, this is a good suggestion. The twig templates have not-quite-identical but a similar idea in the head defining everything that the template accepts: https://github.com/NYPL/nypl-design-system/blob/edad8da424279a9ae7b3b0ec7b722e74e5f8269a/src/twig/_patterns/03-organisms/event-section/_event-section.twig#L1